pkgs/top-level/php-packages.nix

```diff
@@ -617,7 +623,20 @@ in {
           configureFlags = [
             "--enable-simplexml"
           ];
-          patches = lib.optionals (lib.versions.majorMinor php.version == "8.2" && lib.versionOlder php.version "8.2.22") [
+          patches = lib.optionals (lib.versions.majorMinor php.version == "8.1") [
+            (fetchpatch {
```

Comment:
I like the idea of having comments before each patch so we know if we have to keep them or remove them at the next bump. Can you please add a single comment line like you did in pkgs/development/interpreters/php/8.1.nix ?